src/java/org/apache/cassandra/config/EncryptionOptions.java

```diff
@@ -98,7 +99,7 @@ public String description()
      * We will wait to initialize this until applyConfig() call to make sure we do it only when the caller is ready
      * to use this option instance.
      */
-    public ISslContextFactory sslContextFactoryInstance;
+    public transient ISslContextFactory sslContextFactoryInstance;
```

Comment:
`transient` causes this field to be hidden, right now you could try to set it in YAML and will get a weird error as SnakeYAML doesn't know how to construct the interface